### ISSUE TITLE
error occur with stable diffusion latest update

### ISSUE THREAD

**Issue by lilcdj** [Gerschel/sd_web_ui_preset_utils#47]
Hi. I can't save preset because I get error whenever I save new preset.

**Comment by rkolbaskin** [comment]
As a dirty fix - open `extensions/sd_web_ui_preset_utils/scripts/zpreset_utils.py` and comment out lines 529 and 530:
```
529                    elif k == "Sampling method":
530                        return_dict.update({k: modules.sd_samplers.samplers[new_setting[i]].name if self.is_txt2img else  modules.sd_samplers.samplers_for_img2img[new_setting[i]].name})
```

**Comment by merecesarchviz** [comment]
i have the same issue!

I comment the lines 529 and 530 but still give errors and dont save it!

**Comment by rkolbaskin** [comment]
@merecesarchviz You did restart SD, right?
Could you attach traceback?

**Comment by merecesarchviz** [comment]
> @merecesarchviz You did restart SD, right? Could you attach traceback?

yes i restart it, here is the errors now after comment that 2 lines:

![image](https://github.com/Gerschel/sd_web_ui_preset_utils/assets/79140456/6b8010ca-9f98-4dcc-b4b9-1620fdae64f6)

**Comment by rkolbaskin** [comment]
@merecesarchviz it looks like you are using a different save button (I only used the one in "Quick" tab). Try to also comment out 578 and 579. They have basically the same code in there.

**Comment by merecesarchviz** [comment]
yes it was that, that for the tips!

**Comment by cnsit** [comment]
For those are troubled by this issue, a quick work around before the fix: comment out line 529, 530, 578, 579 in the file "extensions/sd_web_ui_preset_utils/scripts/zpreset_utils.py", and then restart your SD-UI.
